tests/devapps/RegionalTestApp/Program.cs

```diff
@@ -45,7 +45,7 @@ static async Task Main(string[] args)
 
             Console.WriteLine("=== Acquire token global ===");
             await AcquireTokenAsync(certificate, false).ConfigureAwait(false);
-
+            Console.ReadLine();
```

Comment:
What is the reason for this?

Reply:
This was to prevent the console window from closing when running this dev app executable (like on a test VM).